labrad/backend.py

```diff
@@ -45,6 +47,13 @@ def disconnect(self):
             self._disconnect()
             self.connected = False
 
+    def spawn(self):
+        """Start a new independent backend connection to the same manager."""
+        inst = self.__class__(name=self.name)
```

Comment:
Does this spawn a new instance of the class? I don't understand why you're directly calling the ```__class__``` method.